version.txt

```diff
@@ -0,0 +1 @@
+0.11.0.0
```

Comment:
Shouldn't be `1.0.0.0` since this PR targets `dev/1.0.0`?

Reply:
Fixed